### ISSUE TITLE
Rework the pet system.

### ISSUE THREAD

**Issue by synoptlcal** [DFS-Servers/BugReportsAndSuggestions#297]
**Describe your suggestion**
As of now pets serve no purpose other than for trading and showing off the unusual effects worn on them.
Only things you can really do to interact with your pet is feeding them, sending them out and other utility options (removing the color from them as an example)
Pets also have a few bugs that also persisted for a while, for example pets follow certain players that the pet owner currently spectates(?)
This system persisted for a long while and at the same time it's necessary to download all pet related customs, with no option of disabling pets.
tl;dr: **right now, pets are purely cosmetic and take up download time. Pets need to be given a wider purpose and more options to interact with them.**

**Describe alternatives you've considered**
Developing a unique system for pets can be very hard especially without affecting combat elements.
My honest take for pets, even if it's a drastic solution, is to simply remove them _if_ there's barely any possible merits for them other than boosting EXP and credit drops which doesn't help a lot to make pets more useful.

**Comment by Light08** [comment]
Pet 2.0 rework has been planned for a while which will breath more life into pets.
No ETA. No need to make this post since it's already eventually planned once we get the opportunity.
